.github/workflows/alpha.yml

```diff
@@ -60,4 +60,75 @@ jobs:
\ No newline at end of file
+        sz deploy web-app --stage alpha --message "Workflow $GITHUB_JOB, commit $GITHUB_SHA" --credentials sharezone-prod-key.json
+
+  deploy-alpha-android-app:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Set Flutter version from FVM config file to environment variables
+      uses: kuhnroyal/flutter-fvm-config-action@v1
+
+    - name: Install Flutter
+      uses: subosito/flutter-action@v2
+      with:
+        flutter-version: ${{ env.FLUTTER_VERSION }}
+        channel: ${{ env.FLUTTER_CHANNEL }}
+
+    - name: Setup signing
+      working-directory: app/android
+      env:
+        KEYSTORE_STRING: ${{ secrets.SHAREZONE_ANDROID_APP_KEYSTORE_STRING }}
+        KEY_PASSWORD: ${{ secrets.SHAREZONE_ANDROID_APP_KEY_PASSWORD }}
+        KEY_ALIAS: ${{ secrets.SHAREZONE_ANDROID_APP_KEY_ALIAS }}
+        STORE_PASSWORD: ${{ secrets.SHAREZONE_ANDROID_APP_STORE_PASSWORD }}
+      run: |
+        echo $KEYSTORE_STRING | base64 -di > app/key.jks
+        echo "storePassword=$STORE_PASSWORD" >> key.properties
+        echo "keyPassword=$KEY_PASSWORD" >> key.properties
+        echo "keyAlias=$KEY_ALIAS" >> key.properties
+        echo "storeFile=key.jks" >> key.properties
+    
+    - name: Build Android
+      working-directory: app
+      run: | 
+        # We are publishing APKs instead of App Bundles to Firebase Distribution
+        # because they easier to install. App Bundles are installed via the
+        # PlayStore which resulted to problems in the fast.
+
+        flutter build apk \
+          --release \
+          --flavor prod \
+          --dart-define DEVELOPMENT_STAGE=ALPHA
+
+    - name: Install Firebase CLI
+      run: sudo npm i -g firebase-tools
+
+    - name: Publish to Firebase Distribution
+      working-directory: app
+      env:
+        FIREBASE_DISTRIBUTION_KEY: ${{ secrets.GCP_SA_FIREBASE_DISTRIBUTION_PROD_KEY }}
+      run: |
+        # Set up credentials for Firebase Distribution
+        echo $FIREBASE_DISTRIBUTION_KEY > sharezone-prod-key.json
+        export GOOGLE_APPLICATION_CREDENTIALS=sharezone-prod-key.json
+        
+        # Because we are publishing every commit a new alpha version, we are
+        # able to use the last commit message (title and description) as release
+        # note for the alpha builds. This is not the most user friendly note but
+        # it's better than nothing.
+        export LAST_COMMIT_MESSAGE=$(git log -1 --pretty=%B)
+        
+        firebase appdistribution:distribute build/app/outputs/flutter-apk/app-prod-release.apk \
+          --app 1:730263787697:android:f0a3b3b856fd1383 \
+          --groups "alpha" \
+          --release-notes "$LAST_COMMIT_MESSAGE"
+
+    - name: Remove credentials
+      if: always()
+      run: |
+        rm app/sharezone-prod-key.json
+        rm app/android/key.properties
+        rm app/android/app/key.jks
```

Comment:
Hm, okay. Can you create an issue for marking them as secrets? (I'm currently on mobile) Just so that we don't forget:)

Reply:
I opened #245 but I think the in the section "Register all secrets used within workflows" they are talking about something different. I understand that if we create a **new** secret within a workflow (like generating a JWT), you can mark this as secret so that this secret with not shown in the logs. But all the files that we created like `sharzeone-prod-key.json`, `key.properties`, etc. are just already registered secrets (just written in a file). So they should already be masked in the logs.

My concern was a different one. I was more thinking about that the company GitHub is compromised and someone got access to the GitHub Actions containers and is able to read all the files. Or we say that we don't trust GitHub that they really delete all files after finishing a workflow. I know that are rare cases but my idea was to just decrease the risk where we have the impact to do it (especially when it's so easy like here we just execute 3 commands).